lib/taskflows/copy-source-task.js

```diff
@@ -12,8 +13,12 @@ class CopySourceTask extends Task {
     this.excludeFiles = excludeFiles;
   }
 
-  async run() {
-    await copySourceTo(this.sourceDir, this.targetDir, this.excludeFiles);
+  async run() {    
+    if (this.sourceDir === this.targetDir) {
+      log.debug('skipping copies, this.sourceDir %s, this.targetDir %s', this.sourceDir, this.targetDir); 
```

Comment:
Skipping copy， the source dir is %s, the target dir is %s

Reply:
done